app/controllers/application_controller/current_user.rb

```diff
@@ -30,7 +30,7 @@ def super_admin_user?
 
   def current_user
     if current_userid
-      @current_user ||= User.find_by_userid(current_userid).tap do |u|
+      @current_user ||= User.find_by(:userid => current_userid).tap do |u|
```

Comment:
Not sure if this change is valid.

`User.find_by_userid` is an actual method that does `User.in_my_region.find_by(:userid ...)`

Reply:
You're right. I'm closing this PR.